geopandas/base.py

```diff
@@ -43,8 +43,9 @@ def _binary_geo(op, left, right):
         return GeoSeries(data, index=left.index, crs=left.crs)
 
     elif isinstance(right, BaseGeometry):
-        data = np.array([getattr(s, op)(right) for s in left.geometry],
-                        dtype=object)
+        # ensure 1D output, see note above
```

Comment:
we actually already did exactly this ten lines above this one in the code path where `right` is a GeoSeries/GeoDataFrame